src/image.ts

```diff
@@ -65,7 +65,11 @@ export class Image extends Construct {
     super(scope, id);
     const registry = props.registry ?? 'docker.io/library';
     const tag = `${registry}/${Names.toDnsLabel(this)}`;
-    const buildArgs = props.buildArgs?.map(arg => `--build-arg ${arg.name}=${arg.value}`) ?? [];
+    let buildArgs;
```

Comment:
Line 74 causes problems if no buildArgs are specified since `...buildArgs` will throw an error. We need to initialize the array first. (Also, using const is fine here since the array itself is not being changed, only its contents).
```suggestion
    const buildArgs = [];
```